### PR TITLE
[3.x] Disable checkout button and add notice if not in stock

### PR DESCRIPTION
**Pull request by BobWez98**
Also see depending on:
https://github.com/rapidez/core/pull/627
https://github.com/rapidez/core/pull/628 

Internally there's a story to have some extra frontend added for this.

V2: #127

### CODE REVIEW

**Review by Roene-JustBetter — APPROVED** [review]

File: resources/views/cart/partials/sidebar/summary.blade.php [hunk 1]
```diff
@@ -29,12 +29,17 @@
...
         </div>
     </x-rapidez-ct::separated-listing>
-
-    <x-rapidez-ct::button.enhanced :href="route('checkout')" class="flex w-full items-center justify-center gap-1 mt-6" dusk="checkout">
-        @lang('To checkout')
-        <x-heroicon-o-arrow-right class="h-4" />
-    </x-rapidez-ct::button.enhanced>
-
+    <div class="w-full" :class="{ 'cursor-not-allowed': !canOrder }">
+        <x-rapidez-ct::button.enhanced
```
[the remark this answers is not in the document]
Comment (BobWez98) [reply resources/views/cart/partials/sidebar/summary.blade.php:33]: As this button has a href, this is a `<a href` and not a button. The disabled attribute does not work for a link.

[the remark this answers is not in the document]
Comment (jordythevulder) [reply resources/views/cart/partials/sidebar/summary.blade.php:33]: You're right!

[the remark this answers is not in the document]
Comment (jordythevulder) [reply resources/views/cart/partials/sidebar/summary.blade.php:33]: And what if we replaced the "continue" button with a button that removed the "Out of stock" products. "Remove out of stock products" for example. Or doesn't that functionality cover the scope of this user story?

[the remark this answers is not in the document]
Comment (BobWez98) [reply resources/views/cart/partials/sidebar/summary.blade.php:33]: Good idea, create a story for it.

**Review by JimmyHoenderdaal — APPROVED** [review]

**Review by jordythevulder — APPROVED** [review]

**Comment by royduin** [comment]
Seems like this requires something from the core that isn't merged yet? This package should require the correct core version when that's merged and released.

**Comment by royduin** [comment]
This has been reverted, please arrange a follow-up with the right version constraint

**Pull request merged**